src/instancemanager.cpp

```diff
@@ -120,7 +120,7 @@ QString InstanceManager::manageInstances(const QStringList &instanceList) const
 				if (!deleteLocalInstance(choice))
 				{
 					QMessageBox::warning(nullptr, QObject::tr("Failed to delete Instance"),
-						QObject::tr("Could not delete Instance \"%1\"").arg(choice), QMessageBox::Ok);
+						QObject::tr("Could not delete Instance \"%1\". \nIf the folder was still in use, restart MO and try again.").arg(choice), QMessageBox::Ok);
```

Comment:
I would change that:
``Could not delete Instance \"%1\". \nIf the folder was still in use, restart MO and try again.``
to something more like that:
``Could not delete Instance \"%1\". \nThe folder is probably used by some other program, restart MO and try it again.``
To emphasize that some program is locking the instance folder/sub-folder.

Reply:
MO is the program that is still using the instance. This happens when somebody clicks on the change instance icon and then proceeds to delete the instance he just came from.

Reply:
Also I put if because that same dialogue is shown in whatever case the delete was not successful, even if the user cancelled the delete dialog for example

Reply:
Then I would prefer this:
``Could not delete Instance \"%1\". \nThe folder is still being used, restart MO and try it again.``
Your original text suggests that the folder is maybe in use and could confuse end-user.

Reply:
But it is as I wrote, it could be in use, or the user might just just have cancelled the "Are you sure you want to delete this folder" message. The same error will appear. Anything could have stopped that folder from deleting, it's just that if the cause was "the folder is in use" as windows will tell the user then they have to restart MO to make it work.

Reply:
Please look into what we are talking about first.

Reply:
If you cancel the dialog no error should occur at all except that the user cancelled the dialog. Now the error message gives the user no real hint why it didn't work. If he dismissed the dialog by accident he still just think that this feature does not work and report it to us.

Reply:
Please just try it out first to understand what the actual dynamic is.